src/Entity/Type.php

```diff
@@ -5,16 +5,21 @@
 use DateTime;
 use DateTimeInterface;
 use Doctrine\ORM\Mapping as ORM;
+use JMS\Serializer\Annotation\Groups;
+use JMS\Serializer\Annotation as Serializer;
 
 /**
  * @ORM\Entity(repositoryClass="App\Repository\TypeRepository")
+ * @Serializer\ExclusionPolicy("all")
  */
 class Type
 {
     /**
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @Serializer\Expose()
+     * @Groups({"ActivityDetails"})
```

Comment:
Why have you made only this field available in the ActivityDetails? What about the others?